demo/CMakeLists.txt

```diff
@@ -1,18 +1,17 @@
 cmake_minimum_required (VERSION 3.1)
-project (PegasDemo)
+project (PegasusDemo)
 
-set (CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR})
 
 #########################################################
-# FIND GLUT
+# INCLUDE FreeGLUT
 #########################################################
-find_package(GLUT REQUIRED)
-include_directories(${GLUT_INCLUDE_DIRS} ${GLUT_INCLUDE_DIR})
-link_directories(${GLUT_LIBRARY_DIRS})
-add_definitions(${GLUT_DEFINITIONS})
-if(NOT GLUT_FOUND)
-    message(ERROR " GLUT not found!")
-endif(NOT GLUT_FOUND)
+set(GLUT_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/FreeGLUT/freeglut/freeglut")
+set(GLUT_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/third_party/FreeGLUT/freeglut/freeglut")
```

Comment:
Should not rely on `${CMAKE_CURRENT_SOURCE_DIR}` but instead to some `${PEGASUS_ROOT}`
Also it should not be here, but in `GlutConfig.cmake` instead